hack/helm/pre_helm.go

```diff
@@ -2,12 +2,16 @@ package main
 
 import (
 	"bytes"
-	"io/ioutil"
```

Comment:
why `io/ioutil` is replaced with the `io` package in all places? is it related to linter?

Reply:
Yeah.` io/ioutil` is deprecated